scripts/test_reproducibility.py

```
@@ -0,0 +1,90 @@
+    all_match = all(h == hashes[0] for h in hashes)
+    return all_match, hashes, hashes[0] if hashes else ""
+
+
+def run_all_tests(runs: int, seed: int) -> Tuple[Dict[str, bool], str]:
+    print("=" * 60)
+    print("Running Reproducibility Tests")
+    print("=" * 60)
+    results = {}
+    
+    print("\n1. Testing reproducibility across multiple runs...")
+    success, hashes, final_hash = _reproducibility_multiple_runs(runs, seed)
```

Ruff (0.15.2)

[warning] Unpacked variable `hashes` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)